src/docfx/lib/markdown/MarkdigUtility.cs

```diff
@@ -112,6 +144,102 @@ public static MarkdownPipelineBuilder Use(this MarkdownPipelineBuilder builder,
             return builder;
         }
 
+        public static string? GetElementType(MarkdownObject node)
+        {
+            if (node is null)
+            {
+                return default;
+            }
+
+            string? elementType = s_markdownElementTypeMapping.GetValueOrDefault(node.GetType(), null);
+            if (elementType is null)
+            {
+                if (node is HeadingBlock heading)
```

Comment:
Is this a perfect place for [switch expression](https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#switch-expressions)?